src/metabase/driver/sql/parameters/substitute.clj

```diff
@@ -95,7 +95,7 @@
                                         e))))]
     (log/tracef "=>%s\n%s" sql (pr-str args))
     (when (seq missing)
-      (throw (ex-info (tru "Cannot run the query: missing required parameters: {0}" (set missing))
-               {:type    qp.error-type/missing-required-parameter
-                :missing missing})))
+      (throw (ex-info (tru "Something something, this seems to be returned as json")
```

Comment:
I changed this to use the same error-type I need to see if that was the problem, but this does get returned as a json from the endpoint.